### DISCUSSION TITLE
Gdevelop counts entire image as hitbox

### DISCUSSION THREAD

**Discussion by VATstudios** [4ian/GDevelop#2361] [Enhancement ideas (and their technical discussions)]
Hey, I have a huge map image (10000*10000) that I want to use as a platform. It has blank pixels between all the actual platforms, but gdevelop counts the ENTIRE image as one hitbox. Is there an easy fix for this? It would take hours for me to manually input the hitboxes.

**Comment by 4ian** [comment]
This image is huge! We should probably add a warning in GDevelop that it is not recommended at all to use such a large image, it will easily destroy performance on the less capable GPUs/mobile phones: https://trello.com/c/VY5E4em6/204-warn-the-user-when-an-image-too-large-is-used-which-could-decrease-performance-a-lot

I would strongly suggest to either:
- cut your image into reusable chunk (platforms, etc...) that you can reuse. You can customize the hitbox of each.
- or cut your image into 4 or 16 images, and put on top of it invisible objects that act as platform - that you hide with events.

In the first solution, you design your level in GDevelop. In the second solution, you design and export your level into multiple images from another software, but you still have to create the hitboxes using invisible objects (that can be resized, etc.. using the scene editor).
